14/umbraco-cms/reference/security/two-factor-authentication.md

```diff
@@ -206,11 +206,12 @@ using Google.Authenticator;
 using Umbraco.Cms.Core.Models.Membership;
 using Umbraco.Cms.Core.Security;
 using Umbraco.Cms.Core.Services;
+using Umbraco.Extensions;
```

Comment:
it also shows that this not necessary to be added as is it not used anywhere 🤔 
![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/b06e08d0-d85a-4051-a728-ff404e81785b)

Reply:
I think you are right. That can go away. My IDE messed up!